### ISSUE TITLE
onnx op prim::Undefined()

### ISSUE THREAD

**Issue by wangyunxiaa** [onnx/onnx#1523]
pytorch to onnx
the converted onnx model has some op like"prim::Undefined()", and the source code in onnx about the describe is 

'''Undefined is not an ONNX operator; keep it as prim::Undefined
and let the exporter handle finally eliminating these'''

elif op_name == "Undefined":
     return None

when i print my model, i got the following result:
when i do "torch.onnx.export" ,the printed model struct is:

%69 : Float(3, 10001) = onnx::LogSoftmax[axis=1](%66), scope: RNNModel/IndexLinear[criterion]/CrossEntropyLoss[ce]
%70 : Dynamic = prim::Undefined(), scope: RNNModel/IndexLinear[criterion]/CrossEntropyLoss[ce] 
%71 : Float(3) = onnx::ATen[ignore_index=-100, operator="nll_loss", reduction=0](%69, %68, %70), scope: RNNModel/IndexLinear[criterion]/CrossEntropyLoss[ce]

and when i check the onnx model,the model struct is:

%69 = LogSoftmax [axis = 1] (%66)
%71 = ATen[ignore_index = -100, operator = 'nll_loss', reduction = 0](%69, %68, %)

the %70 op is Dynamic = prim::Undefined(),and when the model converted to onnx,the op is disappear, the op %71 missing one parameter of %70,
I want to know what the "prim::Undefined()" mean,and if the model has this type op, does these op affect the subsequent conversion?

**Comment by prasanthpul** [comment]
torch.onnx.export is part of PyTorch. Please file the issue in the PyTorch GitHub.
